python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -215,6 +215,23 @@ def create_api_shape_node(tensor_shape_node):
     return api_shape_node
 
 
+def get_constant_variable_node(name, value, shape=[1], dtype='int64'):
+    return gast.parse('%s = fluid.layers.fill_constant(%s, "%s", %s)' %
+                      (name, str(shape), dtype, str(value)))
+
+
+def get_attribute_full_name(node):
+    assert isinstance(
+        node,
+        gast.Attribute), "Input non-Attribute node to get attribute full name"
+
+    if isinstance(node.value, gast.Name):
+        return node.value.id + "." + node.attr
+    elif isinstance(node.value, gast.Constant):
+        return str(node.value.value) + "." + node.attr
+    return get_attribute_full_name(node.value) + "." + node.attr
+
+
```

Comment:
One question:
Can this function `get_attribute_full_name` be replaced by it:
```
def get_attribute_full_name(node):
    assert isinstance()
    return astor.to_source(gast.gast_to_ast(node)).strip()
```

Reply:
Sure